assets/javascripts/main.js

```diff
@@ -47,9 +47,20 @@
     });
   }
 
+  function initHighlights() {
+    $('.column-value, .column-title').on('mouseover', function() {
+      var cointainer = $(this).parents('.container');
+      cointainer.find('.column-value').removeClass('highlight')
```

Comment:
This `find` method of jQuery returns a collection of elements, and you can use selector over this collection with jQuery `filter` method and it is possible to use jQuery's `merge` for joining collections.
It should reduce complexity and improve readability and performance (as dom searches does not fast).

What do you think about this refactor:
```js
var cointainer = $(this).parents('.container'),
    cells      = $.merge(cointainer.find('.column-value'), cointainer.find('.column-title')),
    childIndex = $(this).index() + 1;

cells.removeClass('highlight');
cells.filter('nth-child('+ childIndex +')').addClass('highlight');
```

Reply:
Awesome, thanks!